cloud-sql-proxy/cloud-sql-proxy.sh

```diff
@@ -21,8 +21,21 @@
 # Do not use "set -x" to avoid printing passwords in clear in the logs
 set -euo pipefail
 
-readonly ADDITIONAL_INSTANCES_KEY='attributes/additional-cloud-sql-instances'
+declare -A DEFAULT_DB_PORT=(['MYSQL']='3306' ['POSTGRES']='5432' ['SQL']='1433')
```

Comment:
Is it possible to do a conversion, replace SQL with SQLSERVER?

Reply:
good catch! I just created one sqlserver instance, the instance type is actually SQLSERVER. updated relevant info